tests/acceptance/test_project_settings_sampling.py

```diff
@@ -6,7 +6,7 @@
 from sentry.models import ProjectOption
 from sentry.testutils import AcceptanceTestCase
 
-FEATURE_NAME = "organizations:server-side-sampling"
+FEATURE_NAME = ["organizations:server-side-sampling" + "organizations:server-side-sampling-ui"]
```

Comment:
this concatenates two strings, i don't think that's what you want

Reply:
thanks! let me fix that and move it to a separate PR so that we don't have frontend and backend tags on this one
https://github.com/getsentry/sentry/pull/37595